### PR TITLE
Add page progression direction feature

### PR DESCRIPTION
**Pull request by 1l0**
Typically, this adds rtl(right to left) feature for ```writing-mode: vertical-rl;``` in css.

### CODE REVIEW

File: pkg.go [hunk 1]
```diff
@@ -115,6 +115,7 @@ type pkgMetadata struct {
 type pkgSpine struct {
 	Items []pkgItemref `xml:"itemref"`
 	Toc   string       `xml:"toc,attr"`
+	Ppd   string       `xml:"page-progression-direction,attr"`
```
Comment (bmaupin) [inline pkg.go]: Could you use `omitempty` here (https://github.com/bmaupin/go-epub/blob/master/pkg.go#L81)? I think that should prevent this from being added if it's not set, which should also keep from breaking TestEpubWrite (https://travis-ci.org/bmaupin/go-epub/builds/183031639)

File: epub.go [hunk 2]
```diff
@@ -75,6 +75,7 @@ img {
 	defaultCoverImgFormat     = "cover%s"
 	defaultCoverXhtmlFilename = "cover.xhtml"
 	defaultEpubLang           = "en"
+	defaultPpd                = "ltr"
```
Comment (bmaupin) [inline epub.go]: I wonder if "default" would be a better default value based on the spec. What do you think?:

> When the default value is specified, the Author is expressing no preference and the Reading System may chose the rendering direction. This value must be assumed when the attribute is not specified.
http://www.idpf.org/epub/301/spec/epub-publications.html#sec-spine-elem

File: epub.go [hunk 3]
```diff
@@ -352,6 +356,11 @@ func (e *Epub) SetLang(lang string) {
 	e.pkg.setLang(lang)
 }
 
+func (e *Epub) SetPpd(direction string) {
```
Comment (bmaupin) [inline epub.go]: Would you be able to add a test for this new method?

File: epub.go [hunk 4]
```diff
@@ -356,6 +356,7 @@ func (e *Epub) SetLang(lang string) {
 	e.pkg.setLang(lang)
 }
 
+// SetPpd sets the page progression direction of the EPUB.
 func (e *Epub) SetPpd(direction string) {
```
Comment (bmaupin) [inline epub.go]: Would you be able to add a test for this new method?

File: pkg.go (hunk 1 above)
Comment (1l0) [inline pkg.go]: Sure.

File: epub.go (hunk 2 above)
Comment (1l0) [inline epub.go]: Absolutely. I think defaultPpd is not needed anymore though.

File: epub.go (hunk 4 above)
Comment (1l0) [inline epub.go]: Is this ok?

File: epub.go (hunk 2 above)
Comment (bmaupin) [inline epub.go]: Yeah, I think you're right.

**Comment by bmaupin** [comment]
Looks great. I'll try to merge it soon and I'll remove defaultPpd after that.

Thanks for the contributions!

**Comment by 1l0** [comment]
My pleasure, thanks for the review.

**Pull request merged**